deploy/install-driver.sh

```diff
@@ -16,6 +16,13 @@
 
 set -euo pipefail
 
+# min version of this nfs plugin supporting CSI snapshots
+snap_ver='4.3.0'
+
+# external-snapshotter to install for snapshots to work
+snap_controller_repo='github.com/kubernetes-csi/external-snapshotter/deploy/kubernetes/snapshot-controller?ref=v6.2.1'
```

Comment:
I think we need to install snapshot controller in charts, also that's configurable

Reply:
good point, I will include that along with the unit test coverage in this PR later this week when time permits. Pitty https://github.com/kubernetes-csi/external-snapshotter/issues/812 is not yet done as that would make the dependency tracking for helm a lot easier.

Reply:
thanks, here is an example of snapshot controller chart config in csi driver charts: https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/8b37cd361590a79f23239cc2fb74f90cf58bd667/charts/latest/azurefile-csi-driver/values.yaml#L131-L155

Reply:
embedded external-snapshotter resources to the helm chart but I am unsure about https://github.com/kubernetes-csi/csi-driver-nfs/pull/430#discussion_r1148641925

Reply:
could you copy the snapshot-controller deployment and CRDs from charts into `deploy` folder?